lib/routes/api/speech-credentials.js

```diff
@@ -147,19 +159,29 @@ router.get('/:sid', async(req, res) => {
     if (0 === cred.length) return res.sendStatus(404);
     const {credential, ...obj} = cred[0];
     if ('google' === obj.vendor) {
-      obj.service_key = decrypt(credential);
+      const o = JSON.parse(decrypt(credential));
+      const key_header = '-----BEGIN PRIVATE KEY-----\n';
+      const obscured = {
+        ...o,
+        private_key: `${key_header}${obscureKey(o.private_key.slice(key_header.length, o.private_key.length))}`
+      };
+      obj.service_key = JSON.stringify(obscured);
     }
     else if ('aws' === obj.vendor) {
       const o = JSON.parse(decrypt(credential));
-      obj.access_key_id = o.access_key_id;
-      obj.secret_access_key = o.secret_access_key;
+      obj.access_key_id = obscureKey(o.access_key_id);
```

Comment:
I guess I was wrong here since AWS doesn't obscure this in their web UI we probably don't need to be concerned about it either 🙃 . sorry.

Reply:
So the access key isn't obscured? Sure, it will be a quick fix if that is.